tasks/main.yml

```diff
@@ -170,6 +176,12 @@
     or __hpc_versionlock_content.stdout is not search(item  + "-[0-9]")
   loop: "{{ __hpc_versionlock_rpms }}"
 
+- name: Update all packages to bring system to the latest state
```

Comment:
this needs to happen at the very beginning of the process, before you do any version lock or any installation of anything related to HPC.